ansible/tasks/api/01_checkauth.yml

```diff
@@ -5,4 +5,4 @@
   register: queuesnoauth
 - name: Check if authentication works
   assert:
-    that: "'{\"errors\": {\"Authentication Required\": \"Please provide valid Token verification credentials\"}}' in queuesnoauth.content"
+    that: "'{\"error\":{\"Authentication Required\":\"Please provide valid Token verification credentials\"}}' in queuesnoauth.content"
```

Comment:
If the response changed, that indicates a breaking change.

Reply:
Thanks for checking all the details! I agree it's OK to go with the change then, but it should be mentioned in the CHANGELOG of intelmq-api